languages-and-frameworks/wordpress.html.markerb

```diff
@@ -0,0 +1,160 @@
+---
+title: Run a Wordpress app
```

Comment:
Use WordPress here and throughout since that's how they style the word.

Also capitalize "App" (trying to move to sentence case headings but the frameworks section uses the heading for the sidenav so we need to be consistent with the rest)